### PR TITLE
CYBERRU-322 split DotaMatchSummary into multiple structures

### PR DESCRIPTION
**Pull request by kurenkoff**

### CODE REVIEW

File: structs/dota_match_summary.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+// DotaDraftEvent is information about event of draft phase of Dota match.
+type DotaDraftEvent struct {
+	Order    int64  `json:"order"`
+	Type     string `json:"type"`
+	RosterId int64  `json:"roster_id"`
+	Hero     Hero   `json:"hero"`
+}
+
+// RoshanEvent is a summarization of data about Roshan event of Dota match.
```
Comment (patstrom) [inline structs/dota_match_summary.go]: This seems to be the documentation for the RoshanEvent struct

File: structs/dota_match_summary.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	Victim  int64   `json:"victim"`
+	AtTime  int64   `json:"at_time"`
+	Assists []int64 `json:"assists"`
+}
+
+// DotaStructureDest is a summarization of data about structure destruction event of Dota match.
+type DotaStructureDest struct {
+	Killer        int64  `json:"killer"`
+	StructureType string `json:"structure_type"`
+	StricturePos  string `json:"structure_pos"`
```
Comment (patstrom) [inline structs/dota_match_summary.go]: `StricturePos` should probably be `StructurePos`.

**Review by patstrom — CHANGES_REQUESTED** [review]
Thank you for your PR! And thank you for fixing all the typos of "structure" that was in the old version. Something must've gone terribly wrong when auto-generating those.

Just a few typos and copy-paste errors to fix and I'm happy about this.

File: structs/dota_match_summary.go (hunk 1 above)
Comment (kurenkoff) [reply structs/dota_match_summary.go]: Ye, funny copy-paste error

**Comment by kurenkoff** [comment]
@patstrom Hey, thank you for your feedback. I've fixed errors and also added some fields which were declared in API Reference but not presented in go code (networth, camps_staked, wards, and runes)

**Review by patstrom — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**